src/App.vue

```diff
@@ -94,7 +94,7 @@
                 @fetchError="fetchErrorDialog = true"
                 @haveData="haveData = true"
                 @filingData="filingData = $event"
-                @haveChanges="showFeeSummary = $event"
+                @haveChanges="stateChangeHandler($event)"
```

Comment:
OK for now but we've been talking about getting of this event in favour of store flags. Maybe.

Reply:
kk, copy that. Agile!

Reply:
Future...